admin/coverage

```diff
@@ -4,18 +4,24 @@ set -e
 
 MAIN_CRATE=portable-rustls
 
+ALL_FEATURES_EXCEPT_UNSAFE=$(admin/all-features-except unsafe-assume-single-core $MAIN_CRATE)
+
+# also need to exclude `unstable-api-all` in both of these cases as it includes both `brotli` & `zlib`
+ALL_FEATURES_EXCEPT_UNSAFE_AND_BROTLI=$(admin/all-features-except unsafe-assume-single-core,unstable-api-all,brotli $MAIN_CRATE)
+ALL_FEATURES_EXCEPT_UNSAFE_AND_ZLIB=$(admin/all-features-except unsafe-assume-single-core,unstable-api-all,zlib $MAIN_CRATE)
```

Comment:
extra-minor nit: as I said elsewhere these look too long; may be nicer to add another admin script to help shorten these; for example:

```sh
ALL_FEATURES_EXCEPT_UNSAFE_AND_BROTLI=$(admin/all-crate-features-except-unsafe-and brotli)
ALL_FEATURES_EXCEPT_UNSAFE_AND_ZLIB=$(admin/all-crate-features-except-unsafe-and zlib)
```